### PR TITLE
Testing Reversion Speed on Dispute Concluded Against

### PR DESCRIPTION
**Pull request by BradleyOlson64**
When a dispute has concluded against a parablock candidate we then want to revert all relay chain blocks on all forks including that candidate. Previously the signal to revert relay chain blocks was only passed on-chain. This meant at least one block worth of wasted effort from block builders as they waited for the concluded dispute to be seen in a block. This change https://github.com/paritytech/polkadot/pull/6512  created a new off-chain means of notifying chain selection when a dispute concludes. 

The logs added in this PR allow us to get a coarse view as to whether the new off-chain means is giving us our expected performance boost.

### CODE REVIEW

File: node/core/chain-selection/src/tree.rs [hunk 1]
```diff
@@ -380,6 +383,12 @@ pub(crate) fn apply_single_reversion(
 	revert_hash: Hash,
 	revert_number: BlockNumber,
 ) -> Result<(), Error> {
+	gum::trace!(
+		target: LOG_TARGET,
+		?revert_number,
+		?revert_hash,
```
Comment (sandreim) [inline node/core/chain-selection/src/tree.rs:389]: We should use same key as above: `revert_hash` or `block_hash`

**Review by sandreim — APPROVED** [review]

File: node/core/chain-selection/src/tree.rs [hunk 2]
```diff
@@ -359,6 +359,9 @@ fn apply_ancestor_reversions(
 	// of unviability is only heavy on the first log.
 	for revert_number in reversions {
 		let maybe_block_entry = load_ancestor(backend, block_hash, block_number, revert_number)?;
+		if let Some(block_entry) = &maybe_block_entry {
+			gum::trace!(target: LOG_TARGET, ?revert_number, ?block_entry.block_hash, "Block marked as reverted via scraped on-chain reversions");
```
Comment (s0me0ne-unkn0wn) [inline node/core/chain-selection/src/tree.rs:363]: nit: `cargo fmt` does not work for macros, it's not that critical, but it's better to fold the lines manually to fit into 100 chars

**Review by s0me0ne-unkn0wn — APPROVED** [review]

**Comment by BradleyOlson64** [comment]
bot merge

**Comment by BradleyOlson64** [comment]
bot merge

**Pull request merged**